.vib/schema-registry/cypress/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost",
+  "responseTimeout": 30000
+}
```

Comment:
The old approach of Cypress env for `compatibilityLevel` was right, but its value should be defined here.